### PR TITLE
[1LP][RFR] Manual Services/SSUI/Service dialog cleanup

### PR DESCRIPTION
**Pull request by niyazRedhat**

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit 28ae0cd17ab8f2203f25484d19886f592abe9854:

`cfme/tests/generic_objects/test_definitions.py`:
- :heavy_exclamation_mark: Line 72:1: **W293** *blank line contains whitespace*
- :heavy_exclamation_mark: Line 95:1: **W293** *blank line contains whitespace*

`cfme/tests/infrastructure/test_vm_migrate.py`:
- :heavy_exclamation_mark: Line 80:1: **W293** *blank line contains whitespace*

`cfme/tests/services/test_catalog_item.py`:
- :heavy_exclamation_mark: Line 332:1: **W293** *blank line contains whitespace*
- :heavy_exclamation_mark: Line 388:78: **W291** *trailing whitespace*

`cfme/tests/services/test_different_dialogs_in_catalogs.py`:
- :heavy_exclamation_mark: Line 134:1: **W293** *blank line contains whitespace*
- :heavy_exclamation_mark: Line 151:22: **W291** *trailing whitespace*
- :heavy_exclamation_mark: Line 167:1: **W293** *blank line contains whitespace*
- :heavy_exclamation_mark: Line 185:1: **W293** *blank line contains whitespace*
- :heavy_exclamation_mark: Line 203:1: **W293** *blank line contains whitespace*
- :heavy_exclamation_mark: Line 205:16: **W291** *trailing whitespace*
- :heavy_exclamation_mark: Line 220:22: **W291** *trailing whitespace*
- :heavy_exclamation_mark: Line 221:1: **W293** *blank line contains whitespace*
- :heavy_exclamation_mark: Line 239:1: **W293** *blank line contains whitespace*
- :heavy_exclamation_mark: Line 257:1: **W293** *blank line contains whitespace*
- :heavy_exclamation_mark: Line 275:1: **W293** *blank line contains whitespace*
- :heavy_exclamation_mark: Line 293:1: **W293** *blank line contains whitespace*
- :heavy_exclamation_mark: Line 310:1: **W293** *blank line contains whitespace*
- :heavy_exclamation_mark: Line 343:1: **W293** *blank line contains whitespace*
- :heavy_exclamation_mark: Line 361:1: **W293** *blank line contains whitespace*
- :heavy_exclamation_mark: Line 379:1: **W293** *blank line contains whitespace*
- :heavy_exclamation_mark: Line 397:1: **W293** *blank line contains whitespace*
- :heavy_exclamation_mark: Line 415:1: **W293** *blank line contains whitespace*
- :heavy_exclamation_mark: Line 433:1: **W293** *blank line contains whitespace*
- :heavy_exclamation_mark: Line 451:1: **W293** *blank line contains whitespace*
- :heavy_exclamation_mark: Line 484:1: **W293** *blank line contains whitespace*
- :heavy_exclamation_mark: Line 502:1: **W293** *blank line contains whitespace*
- :heavy_exclamation_mark: Line 525:1: **W293** *blank line contains whitespace*

`cfme/tests/services/test_dynamicdd_dialogelement.py`:
- :heavy_exclamation_mark: Line 133:1: **W293** *blank line contains whitespace*
- :heavy_exclamation_mark: Line 151:1: **W293** *blank line contains whitespace*
- :heavy_exclamation_mark: Line 184:1: **W293** *blank line contains whitespace*
- :heavy_exclamation_mark: Line 202:1: **W293** *blank line contains whitespace*
- :heavy_exclamation_mark: Line 221:1: **W293** *blank line contains whitespace*
- :heavy_exclamation_mark: Line 239:1: **W293** *blank line contains whitespace*
- :heavy_exclamation_mark: Line 257:1: **W293** *blank line contains whitespace*
- :heavy_exclamation_mark: Line 275:1: **W293** *blank line contains whitespace*
- :heavy_exclamation_mark: Line 293:1: **W293** *blank line contains whitespace*
- :heavy_exclamation_mark: Line 319:1: **W293** *blank line contains whitespace*
- :heavy_exclamation_mark: Line 340:1: **W293** *blank line contains whitespace*
- :heavy_exclamation_mark: Line 357:1: **W293** *blank line contains whitespace*
- :heavy_exclamation_mark: Line 375:1: **W293** *blank line contains whitespace*
- :heavy_exclamation_mark: Line 393:1: **W293** *blank line contains whitespace*
- :heavy_exclamation_mark: Line 415:1: **W293** *blank line contains whitespace*
- :heavy_exclamation_mark: Line 433:1: **W293** *blank line contains whitespace*
- :heavy_exclamation_mark: Line 450:1: **W293** *blank line contains whitespace*

`cfme/tests/services/test_service_catalogs.py`:
- :heavy_exclamation_mark: Line 208:1: **W293** *blank line contains whitespace*
- :heavy_exclamation_mark: Line 218:62: **W291** *trailing whitespace*
- :heavy_exclamation_mark: Line 227:1: **W293** *blank line contains whitespace*
- :heavy_exclamation_mark: Line 229:16: **W291** *trailing whitespace*
- :heavy_exclamation_mark: Line 245:1: **W293** *blank line contains whitespace*
- :heavy_exclamation_mark: Line 247:16: **W291** *trailing whitespace*
- :heavy_exclamation_mark: Line 263:1: **W293** *blank line contains whitespace*
- :heavy_exclamation_mark: Line 281:1: **W293** *blank line contains whitespace*
- :heavy_exclamation_mark: Line 299:1: **W293** *blank line contains whitespace*
- :heavy_exclamation_mark: Line 336:1: **W293** *blank line contains whitespace*
- :heavy_exclamation_mark: Line 354:1: **W293** *blank line contains whitespace*
- :heavy_exclamation_mark: Line 397:1: **W293** *blank line contains whitespace*
- :heavy_exclamation_mark: Line 415:1: **W293** *blank line contains whitespace*
- :heavy_exclamation_mark: Line 433:1: **W293** *blank line contains whitespace*
- :heavy_exclamation_mark: Line 449:1: **W293** *blank line contains whitespace*
- :heavy_exclamation_mark: Line 467:1: **W293** *blank line contains whitespace*
- :heavy_exclamation_mark: Line 540:1: **W293** *blank line contains whitespace*
- :heavy_exclamation_mark: Line 558:1: **W293** *blank line contains whitespace*

`cfme/tests/ssui/test_ssui_service_catalogs.py`:
- :heavy_exclamation_mark: Line 95:1: **W293** *blank line contains whitespace*
- :heavy_exclamation_mark: Line 113:1: **W293** *blank line contains whitespace*
- :heavy_exclamation_mark: Line 146:1: **W293** *blank line contains whitespace*
- :heavy_exclamation_mark: Line 164:1: **W293** *blank line contains whitespace*
- :heavy_exclamation_mark: Line 197:1: **W293** *blank line contains whitespace*
- :heavy_exclamation_mark: Line 215:1: **W293** *blank line contains whitespace*
- :heavy_exclamation_mark: Line 249:1: **W293** *blank line contains whitespace*
- :heavy_exclamation_mark: Line 268:1: **W293** *blank line contains whitespace*

Hmmm
:scream: You might like to check the lint of your code :point_up: 


**No commit flaws detected**.

*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit e5743f5e0479cbcb0fcb243dc3767bef0fa6df51:

`cfme/tests/services/test_catalog_item.py`:
- :heavy_exclamation_mark: Line 386:78: **W291** *trailing whitespace*

`cfme/tests/services/test_different_dialogs_in_catalogs.py`:
- :heavy_exclamation_mark: Line 151:22: **W291** *trailing whitespace*
- :heavy_exclamation_mark: Line 202:16: **W291** *trailing whitespace*

`cfme/tests/services/test_service_catalogs.py`:
- :heavy_exclamation_mark: Line 222:62: **W291** *trailing whitespace*
- :heavy_exclamation_mark: Line 232:16: **W291** *trailing whitespace*
- :heavy_exclamation_mark: Line 249:16: **W291** *trailing whitespace*

`cfme/tests/ssui/test_ssui_service_catalogs.py`:
- :heavy_exclamation_mark: Line 243:19: **W291** *trailing whitespace*

Hmmm
:scream: You might like to check the lint of your code :point_up: 


**No commit flaws detected**.

*CFME QE Bot*

File: cfme/tests/services/test_dynamicdd_dialogelement.py [hunk 1]
```diff
@@ -116,3 +117,322 @@ def test_dynamicdropdown_dialog(appliance, dialog, catalog):
...
+def test_submit_or_cancelation_btns_in_dd_dialogs_tied_to_a_service_button_should_be_visble():
+    """
+    Polarion:
+        assignee: nansari
+        casecomponent: Services
+        initialEstimate: 1/6h
+        testtype: functional
+        startsin: 5.9
+        tags: service
+        tags: service
```
Comment (jawatts) [inline cfme/tests/services/test_dynamicdd_dialogelement.py]: double `tags` keys here

File: cfme/tests/services/test_myservice.py [hunk 2]
```diff
@@ -68,8 +68,9 @@ def test_retire_service_ui(appliance, context, myservice):
         test_flag: provision
 
     Polarion:
-        assignee: sshveta
+        assignee: nansari
         initialEstimate: 1/4h
+        tags: service
```
Comment (jawatts) [inline cfme/tests/services/test_myservice.py]: **Required** Missing `casecomponent` here

File: cfme/tests/services/test_myservice.py [hunk 3]
```diff
@@ -85,8 +86,9 @@ def test_retire_service_on_date(appliance, context, myservice):
...
```
Comment (jawatts) [inline cfme/tests/services/test_myservice.py]: **Required** Missing `casecomponent` here

File: cfme/tests/services/test_myservice.py [hunk 4]
```diff
@@ -103,8 +105,9 @@ def test_crud_set_ownership_and_edit_tags(appliance, context, myservice):
...
```
Comment (jawatts) [inline cfme/tests/services/test_myservice.py]: **Required** Missing `casecomponent` here

File: cfme/tests/services/test_myservice.py [hunk 5]
```diff
@@ -130,6 +133,7 @@ def test_download_file(appliance, context, needs_firefox, myservice, filetype):
     Polarion:
         assignee: nansari
         initialEstimate: 1/16h
+        tags: service
```
Comment (jawatts) [inline cfme/tests/services/test_myservice.py]: **Required** Missing `casecomponent` here

File: cfme/tests/services/test_myservice.py [hunk 6]
```diff
@@ -142,8 +146,9 @@ def test_service_link(appliance, context, myservice, provider):
     """Tests service link from VM details page(BZ1443772)
 
     Polarion:
-        assignee: sshveta
+        assignee: nansari
         initialEstimate: 1/4h
+        tags: service
```
Comment (jawatts) [inline cfme/tests/services/test_myservice.py]: **Required** Missing `casecomponent` here

File: cfme/tests/services/test_service_catalogs.py [hunk 7]
```diff
@@ -36,8 +36,9 @@ def test_order_catalog_item(appliance, provider, catalog_item, request,
         test_flag: provision
 
     Polarion:
-        assignee: sshveta
+        assignee: nansari
         initialEstimate: 1/4h
+        tags: service
```
Comment (jawatts) [inline cfme/tests/services/test_service_catalogs.py]: **Required** Missing `casecomponent` here

File: cfme/tests/services/test_service_catalogs.py [hunk 8]
```diff
@@ -104,8 +106,9 @@ def test_order_catalog_bundle(appliance, provider, catalog_item, request):
...
```
Comment (jawatts) [inline cfme/tests/services/test_service_catalogs.py]: **Required** Missing `casecomponent` here

File: cfme/tests/services/test_service_catalogs.py [hunk 9]
```diff
@@ -142,6 +145,7 @@ def test_no_template_catalog_item(provider, provisioning, dialog, catalog, appli
     Polarion:
         assignee: nansari
         initialEstimate: 1/8h
+        tags: service
```
Comment (jawatts) [inline cfme/tests/services/test_service_catalogs.py]: **Required** Missing `casecomponent` here

File: cfme/tests/services/test_service_catalogs.py [hunk 10]
```diff
@@ -160,8 +164,9 @@ def test_request_with_orphaned_template(appliance, provider, catalog_item):
         test_flag: provision
 
     Polarion:
-        assignee: sshveta
+        assignee: nansari
         initialEstimate: 1/4h
+        tags: service
```
Comment (jawatts) [inline cfme/tests/services/test_service_catalogs.py]: **Required** Missing `casecomponent` here

File: cfme/tests/services/test_service_catalogs.py [hunk 11]
```diff
@@ -192,3 +197,356 @@ def test_advanced_search_registry_element(request, appliance):
...
+        testSteps:
+            1. Create a catalog item
+            2. Upload custom image
+            3. remove custom image
+            4. Create a catalog  bundle
+            5. Upload a custom image
+            6. Change custom image
+        expectedResults:
+            2. No error seen
```
Comment (jawatts) [inline cfme/tests/services/test_service_catalogs.py]: I believe you need all steps in this case, so:
```
1.
2. No error seen
3.
4.
5. No error seen
```

File: cfme/tests/services/test_service_catalogs.py [hunk 12]
```diff
@@ -192,3 +197,356 @@ def test_advanced_search_registry_element(request, appliance):
...
+        initialEstimate: 1/8h
+        startsin: 5.5
+        tags: service
+        testSteps:
+            1. Create a catalog item
+            2. Select cluster and resource pool and Save
+            3. Remove resource pool from provider
+            4. Edit catalog
+        expectedResults:
+            4. Validation message should be shown
```
Comment (jawatts) [inline cfme/tests/services/test_service_catalogs.py]: same comment above about including all steps

File: cfme/tests/test_manual.py [hunk 13]
```diff
@@ -5164,120 +4848,35 @@ def test_osp_test_creating_multiple_migration_plans_with_same_name():
 
 
 @pytest.mark.manual
-@test_requirements.service
-@pytest.mark.tier(3)
-def test_database_wildcard_should_work_and_be_included_in_the_query():
+@test_requirements.ansible
```
Comment (jawatts) [inline cfme/tests/test_manual.py]: missing `tier` here? or will this be addressed by the FAs?

File: cfme/tests/test_manual.py [hunk 14]
```diff
@@ -20162,103 +18126,33 @@ def test_user_cloud_memory_quota_by_services():
...
     """
+    Requires:
+    C&U enabled Vsphere-6 appliance.
+    Steps:
+    1. Navigate to Datastores [Compute > infrastructure>Datastores]
+    2. Select any available datastore
+    3. Go for utilization graphs [Monitoring > Utilization]
+    4. Check data point on graphs ["Used Disk Space", "Hosts", "VMs"]
+    using drilling operation on the data points
+    5.  check "chart" and "display" options working properly or not
```
Comment (jawatts) [inline cfme/tests/test_manual.py]: this should be in `testSteps`

File: cfme/tests/test_manual.py (hunk 13 above)
Comment (niyazRedhat) [reply cfme/tests/test_manual.py]: This is not my FA, I have not added anything there and actually I have deleted them, I have no idea that this added TCs mean and their tier severity.

File: cfme/tests/test_manual.py (hunk 14 above)
Comment (niyazRedhat) [reply cfme/tests/test_manual.py]: This is not my FA, I have not added anything there and actually I have deleted them

**Comment by dajoRH** [comment]
Would you mind rebasing this Pull Request against latest master, please? :trollface:
*CFME QE Bot*

**Comment by izapolsk** [comment]
@niyazRedhat, LGTM. however, please sign your commit. PRT is broken.

**Pull request merged**